docs/submission/dictionary-overview.md

```diff
@@ -1,58 +1,55 @@
+![Conditional](/assets/submission/dictionary-conditional.svg) attribute indicates:
 
-These core clinical fields are commonly acquired in cohort-based studies and clinical trials and are required to address clinically relevant topics by cross entity analyses, and therefore constitute a critical element in the analysis of diverse ARGO Programs.
+- Field must meet certain conditions, depending on the value of another field.
+- Conditions described in the data dictionary scripts & notes.
 
-As seen in the dictionary, most core fields are ![Required](/assets/submission/dictionary-required.svg) for [clinical data completion](clinical-data-validation-rules), with the exception of some conditional fields that are dependent on the values of other fields. Upon upload, a validation error will occur, such as *"vital_status is a required field"*, if a core field is missing.
+![Required](/assets/submission/dictionary-required.svg) attribute indicates:
 
-If the field is ![Core](/assets/submission/dictionary-core.svg) + ![Conditional](/assets/submission/dictionary-conditional.svg), then it is required for clinical data completion, only if certain conditions are met. The conditions for the permissible value will be described in the notes column for the conditional field. 
+- Field must be filled in the submitted TSV file.
+- When pared with the `Conditional` attribute, the field is only required if conditional requirements are met.
 
-#### Extended Fields
-![Extended](/assets/submission/dictionary-extended.svg) fields are not required for clinical data completion but it is strongly encouraged to provide as many extended fields as possible to help ensure data quality. In most cases, extended fields have a blank attribute, which means they are not required to be submitted. 
+![Core](/assets/submission/dictionary-core.svg) attribute indicates:
 
-If a field is classified as ![Extended](/assets/submission/dictionary-extended.svg) + ![ Conditional](/assets/submission/dictionary-conditional.svg), this means you can provide a value for this field only if the condition in the notes is met.
+- Field is part of the mandatory minimum set of clinical data that must be submitted.
+- The set of core clinical fields were defined by the [Tissue & Clinical Annotations Working Group](http://www.icgc-argo.org/page/84/tissue-clinical-annotation-working-group) which involved regular discussions with members of the working group and ARGO Programs.
+- Core clinical fields are commonly acquired in cohort-based studies and clinical trials and are required to address clinically relevant topics by cross entity analyses, and therefore constitute a critical element in the analysis of diverse ARGO Programs.
 
```

Comment:
With how these bulleted lists are written, like "Core data tier indicates:
some of the bullets are more facts about the data tier/attribute...like these two last paragraphs. 

Maybe they should be outside of the bullets as "info  about core" - or maybe "indicates" is just the wrong word.